superset/sql_parse.py

```diff
@@ -41,6 +42,16 @@
 logger = logging.getLogger(__name__)
 
 
+# Workaround for https://github.com/andialbrecht/sqlparse/issues/652.
+sqlparse.keywords.SQL_REGEX.insert(
```

Comment:
do we not need to replace the erroneous regex?

Reply:
@etr2460 I guess one could replace it though the position could change. This is a pattern I've used elsewhere where I simply increase the priority of the match by inserting it at the top of the list.

Reply:
ah, that makes sense. didn't realize the list was prioritized